typescript/cli/src/submit/submitCommand.ts

```diff
@@ -0,0 +1,51 @@
+import {
+  PopulatedTransaction,
+  PopulatedTransactionSchema,
+} from '@hyperlane-xyz/sdk';
+import { assert } from '@hyperlane-xyz/utils';
+
+import { WriteCommandContext } from '../context/types.js';
+import { readYamlOrJson } from '../utils/files.js';
+
+import { getSubmitterBuilder } from './submit.js';
+
+export async function runSubmit({
+  context,
+  transactionsFilepath,
+}: {
+  context: WriteCommandContext;
+  transactionsFilepath: string;
+}) {
+  const { submissionStrategy, chainMetadata, multiProvider, isDryRun } =
+    context;
+
+  assert(
+    submissionStrategy,
+    `Submission strategy required to submit transactions.
+  Please create a submission strategy, e.g. ./strategy.yaml.`,
```

Comment:
Does the CLI check that file location by default? Not seeing that in the command option definition but the e.g. makes me think it does

Reply:
There is no default– the user is required to pass in a submission strategy path to `-s` (and there is intentionally no interactive prompt)

Any ideas on how I can better emphasize this to the user?

I'm afraid `Submission strategy required to submit transactions.` or `Submission strategy required to submit transactions. Please create a submission strategy.` would not be useful

Reply:
You can provide a link to the example yaml file or you can mention the command to create a new strategy (if there is one)